docs/WIRELESS.md

```diff
@@ -158,6 +158,258 @@ The fact that EVE itself stays out of Firmware upgrade business means that you m
+* For WiFi adapters, the Linux [rfkill subsystem](https://linux.die.net/man/1/rfkill) is used
+  to enable/disable radio communication. This, however, requires that for every attached WiFi
+  adapter there is a corresponding rfkill driver available and loaded into the kernel.
+  Run `rfkill` command from the debug console and look for a table entry with `wlan` TYPE.
+  For some devices it may be necessary to enable certain `*_RFKILL=y` kernel parameters
+  to build their drivers (e.g. `CONFIG_ATH9K_RFKILL=y`).
+* For cellular modems we use QMI and MBIM protocols for management. Both of these protocols
+  provide APIs to enable/disable radio transmission. However, cellular modems that do not
+  support any of these protocols (and likely only AT commands), cannot be managed by EVE
+  and the radio transmission will remain in the modem's default state (which usually is ON).
+* Radio silence only applies to wireless network adapters which are visible to EVE (host OS).
+  Adapters directly assigned to applications are not covered. It is up to those applications
+  to manage the state of radio transmission.
+* The intended radio configuration is obtained from the Local profile server by means of periodic
+  polling. However, because we want to limit the delay between a user turning radio silence ON/OFF
+  and the requested change taking the effect, the polling period is set down to 5 seconds,
+  i.e. substantially less than the default 1 minute period for edge node config retrieval
+  from the controller.
+  Still, expect that it may take several seconds to enable/disable radio silence.
+  Especially with cellular modems and QMI/MBIM protocols there is some latency in response
+  from the modem itself.
+  Users should wait for the Local profile server application to display a confirmation
+  in the UI of a finalized state change (and whether it actually succeeded).
+* If the radio silence is imposed while the edge node reboots, there could be a short window
+  between the boot and EVE microservices starting and applying the persisted state,
+  during which wireless devices might in theory manage to quickly turn on and transmit some
+  signals. This also applies to permanently disabled wireless devices (i.e. without network config).
+  However, this risk is mitigated in EVE quite well. For WiFi cards this is actually completely
+  avoided by using a kernel commandline parameter `rfkill.default_state=0`, to ensure that WiFi
+  radios are initially all disabled and can be turned ON only by an EVE microservice.
+  For cellular modems, however, we rely here on the modem to support configuration persistence.
+  For example, SierraWireless modems `EM7565` and `MC7455`, that were tested and verified with EVE,
+  do support persistence.
+* The last limitation (not really a risk) is that by design the local profile override
+  and the radio silence mode both have to be managed by the same application.
+
+### Radio silence implementation
```

Comment:
Since this is about the implementation, should we move it to pkg/pillar/docs/ somewhere? (even if it also touched the wwan container)

Reply:
OK, moved